mailpoet/assets/js/src/form_editor/components/form_settings/form_placement_options/settings_panels/fixed_bar_settings.tsx

```diff
@@ -77,13 +77,13 @@ export function FixedBarSettings(): JSX.Element {
           <AnimationSettings settingsPlacementKey="fixedBar" />
           <SelectControl
             label={MailPoet.I18n.t('formPlacementDelay')}
-            value={formSettings.formPlacement.fixedBar.delay}
+            value={`${formSettings.formPlacement.fixedBar.delay}`}
             onChange={compose([
               changeFormSettings,
               assocPath('formPlacement.fixedBar.delay', __, formSettings),
             ])}
             options={delayValues.map((delayValue) => ({
-              value: delayValue,
+              value: `${delayValue}`,
```

Comment:
Hmm, for 1. I think it's more correct to have it as a `number`. The fact that we use it in a select option that happens to always require strings (as every HTML attribute) shouldn't probably make us change the original type.

I know... at the moment we do save it also as strings to the DB. But (as in other cases in the form editor) we currently actually use both the real type (in defaults) and the stringified type (when changed in the UI). So I guess it's a bigger topic to unify the types altogether.

Another example is booleans — sometimes we save `true`/`false` to the DB (from defaults), sometimes ``1``/ `''` (from the UI).

And you're right for 2., I wanted to avoid any further refactors. The thing is I want to use the `@types` for automation so I needed to have it done in the codebase in general.